tfhe/examples/fhe_strings/README.md

```diff
@@ -0,0 +1,20 @@
+# fhe_strings
+
```

Comment:
I think you use an interesting method by "converting" a string to a uint in some algorithms (if I understand correctly) it could be worth explaining the approach and the intuition behind that transformation in this readme, a small paragraph could be nice